features/gezag-minderjarige/1.2-minderjarig-niet-overleden.feature

```diff
@@ -0,0 +1,296 @@
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam                             | waarde                    |
+      | type                             | TweehoofdigOuderlijkGezag |
+      | minderjarige.burgerservicenummer | 000000036                 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000012 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000024 |
+
+      Voorbeelden:
+      | geboortedatum      |
+      | morgen - 18 jaar   |
+      | gisteren - 17 jaar |
+      | vandaag - 16 jaar  |
+      | gisteren           |
+      | vandaag            |
+
+    Scenario: de persoon is meerderjarig want is <omschrijving>
+      Gegeven de persoon 'P3' met burgerservicenummer '000000036'
+      * is in Nederland geboren
+      * heeft de volgende geboorte gegevens
+      | geboortedatum (03.10) |
+      | <geboortedatum>       |
+      * heeft persoon 'P1' als ouder1 vanaf de geboorteaangifte
+      * heeft persoon 'P2' als ouder2 vanaf de geboorteaangifte
+      * is ingeschreven in de de BRP
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon geen gezag
+
+      Voorbeelden:
+      | geboortedatum      | omschrijving              |
+      | vandaag - 18 jaar  | vandaag 18 jaar geworden  |
+      | gisteren - 18 jaar | gisteren 18 jaar geworden |
+      | morgen - 19 jaar   | 18 jaar                   |
+      | vandaag - 19 jaar  | 19 jaar                   |
+      | gisteren - 20 jaar | 20 jaar                   |
+
+
+  Regel: Als de geboortedatum van de persoon volledig onbekend is wordt de persoon als meerderjarig beschouwd en is gezag niet van toepassing
+
+    Scenario: geboortedatum is volledig onbekend
+      Gegeven de persoon 'P3' met burgerservicenummer '000000036'
+      * is in Nederland geboren
+      * heeft de volgende geboorte gegevens
+      | geboortedatum (03.10) |
+      | 00000000              |
+      * heeft persoon 'P1' als ouder1 vanaf de geboorteaangifte
+      * heeft persoon 'P2' als ouder2 vanaf de geboorteaangifte
+      * is ingeschreven in de de BRP
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon geen gezag
```

Comment:
Is dit een bewuste keuze geweest? 
Ik kan mij hierbij wel iets voorstellen, ik ga er vanuit dat dit alleen maar voorkomt bij personen die in het buitenland geboren zijn. Misschien moeten we dat nog even navragen voor alle zekerheid.